android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/ConfigurationRegistry.kt

```diff
@@ -696,7 +701,8 @@ constructor(
                       TYPE_REFERENCE_DELIMITER,
                     )
                   val resourceId = listEntryComponent.item.reference.extractLogicalIdUuid()
-                  val listResourceUrlPath = "$resourceKey?$ID=$resourceId&_count=$DEFAULT_COUNT"
+                  val listResourceUrlPath =
+                    "${ResourceType.List.name}?$resourceKey?$ID=$resourceId&_page=${applicationConfig.listSyncConfig.page} &_count= ${applicationConfig.listSyncConfig.count}"
```

Comment:
This appears to be adding functionality to the Non-Proxy code block. This Pagination feature is implemented on the OpenSRP FHIR Gateway https://github.com/onaio/fhir-gateway-extension/issues/29

Also see the corresponding PR https://github.com/onaio/fhir-gateway-extension/pull/32

Reply:
Oooh yeah. Correcting that